tools/cmake/profiles/release.cmake

```diff
@@ -6,36 +6,39 @@ function(mbed_set_profile_options target mbed_toolchain)
     list(APPEND link_options)
 
     if(${mbed_toolchain} STREQUAL "GCC_ARM")
-        list(APPEND c_compile_options
-            "-c"
-            "-Os"
-        )
-        target_compile_options(${target}
-            INTERFACE
-                $<$<COMPILE_LANGUAGE:C>:${c_compile_options}>
+        list(APPEND c_compile_options "-c" "-Os")
+        target_compile_options(
+            ${target} INTERFACE $<$<COMPILE_LANGUAGE:C>:${c_compile_options}>
         )
 
-        list(APPEND cxx_compile_options
+        list(
+            APPEND
+            cxx_compile_options
             "-c"
             "-fno-rtti"
             "-Wvla"
             "-Os"
         )
-        target_compile_options(${target}
-            INTERFACE
-                $<$<COMPILE_LANGUAGE:CXX>:${cxx_compile_options}>
+        target_compile_options(
+            ${target}
```

Comment:
what is the difference in formatting here in this (putting target on the second line), compare to `set()`  here https://github.com/ARMmbed/mbed-os/pull/14011/files#diff-04b1fabd6daf85a88decfec2f6395fb7b0f18a9967b94b20a73859f90a34c713L6 (same line) ?

I am just trying to understand  the rules of the formatter so I can manually do the same style :)